mqtt/mqtt.js

```diff
@@ -85,14 +85,15 @@ function startMqtt() {
     return net.createServer(aedes.handle).listen(config.mqtt_port);
 }
 
-nats.subscribe('channel.*', {'queue':'mqtts'}, function (msg) {
+nats.subscribe('channel.>', {'queue':'mqtts'}, function (msg) {
     var m = message.RawMessage.decode(Buffer.from(msg)),
-        packet;
+        packet, subtopic;
     if (m && m.protocol !== 'mqtt') {
+        subtopic = m.subtopic !== '' ? m.subtopic.replace('.', '/') : ''
```

Comment:
is the subtopic in .xxx.yyy.zzz format?

Reply:
it must to be in xxx.yyy.zzz, fixed on last commit sorry